README.md

```diff
@@ -179,8 +172,9 @@ On Linux you can see the output of the application in Vs Code itself. Add `gdbtt
 ```
 ![GdbTTY](gdbttyvscode.png)
 
-### Documentation
+You can also use these options with `gdbtty`: `xterm`, `gnome-terminal`, `konsole` and `xfce4-terminal`.
 
+### Documentation
 For a more in depth documentation please check the [Superbol Documentation](https://ocamlpro.com/superbol/)
```

Comment:
@ddeclerck BTW that page is not public. What do you think would be a more appropriate page? Have we got one?

Reply:
I guess we only have this for now :
https://get-superbol.com/studio/
(which BTW have some dead links at the bottom...)

Reply:
Ok… I'll put that one for now.